internal/locate/region_request.go

```diff
@@ -264,8 +264,9 @@ type replicaSelector struct {
 // selectorState is the interface of states of the replicaSelector.
 // Here is the main state transition diagram:
 //
-//                                    exceeding maxReplicaAttempt
-//           +-------------------+   || RPC failure && unreachable && no forwarding
+//	                         exceeding maxReplicaAttempt
+//	+-------------------+   || RPC failure && unreachable && no forwarding
```

Comment:
You can see the new formatter breaks the diagrams in the comments. At least this should be reverted.

Reply:
Looks like I only need to format my 3 changed files.